ops/terraform/services/common/s3.tf

```diff
@@ -96,3 +96,70 @@ resource "aws_s3_bucket_policy" "logging" {
 }
 POLICY
 }
+
+resource "aws_s3_bucket" "aws-athena-query-results" {
```

Comment:
Is this actually needed? Seems like these files haven't been modified since 2020.

Reply:
Although the bucket has not been used since 2020, we can still monitor it in the test environment for a couple of days as it contains more than 200 objects.